watertap/unit_models/ion_exchange_0D.py

```diff
@@ -584,7 +635,7 @@ def build(self):
         )
 
         self.Sc = Var(
```

Comment:
```suggestion
        self.N_Sc = Var(
```

Reply:
modified all dimensionless numbers to match this format